packages/core/src/audio/AudioManager.ts

```
@@ -1,3 +1,31 @@
+/**
+ * @internal
+ * 自定义 AudioContext 代理类，处理兼容性问题
+ */
+class CompatAudioContext extends (window.AudioContext || window.webkitAudioContext as typeof AudioContext) {
+  constructor() {
+    super();
+  }
+
+  override decodeAudioData(arrayBuffer: ArrayBuffer, successCallback?: Function, errorCallback?: Function): Promise<AudioBuffer> {
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)